### ISSUE TITLE
WILL WORK ON THIS I SWEAR

### ISSUE THREAD

**Issue by NikolaiT** [NikolaiT/GoogleScraper#124]
REMINDER FOR MYSELF: GET SOME MOTIVATION AND FIX ISSUES AND MAKE SOME STABLE RELEASE!!!

**Comment by puzanov** [comment]
Дай Бог тебе сил для этого проекта, Колян :)

**Comment by botzill** [comment]
:+1:

**Comment by Gldkslfmsd** [comment]
Yes, go on, Nikolai! I'm using your GoogleScraper in my own project, but only for parsing Bing SERP. It would be gorgeous if I was able to use it more.

**Comment by NikolaiT** [comment]
Thanks for the huge support! I did actually some work on it :) The next 6 days, I will spent 4 hours on each day on GoogleScraper. Goals: Make it more stable, get some documentation on readthedocs.org and also support different proxies! And the best of it all: Write tests for it such that the pain finally stops...

**Comment by sergiynikolaev** [comment]
:+1:

**Comment by jojomaquiling** [comment]
:+1:

**Comment by VeeeneX** [comment]
Or make somebody collaborator :)

**Comment by crawldnet** [comment]
Great I'm loving this project, I wish it was 100% percent functional
